v3.0/how-to/deploy/orchestrated/ansible.md

```diff
@@ -457,6 +458,7 @@ TiKV3-2 ansible_host=172.16.10.6 deploy_dir=/data2/deploy tikv_port=20172 labels
 
 ......
 
+# Note: For labels in TiKV to work, you must also configure location-labels for PD when deploying the cluster.
```

Comment:
> > `location_labels` in Ansible deployment, while `location-labels` in PD.
> 
> As confirmed with @nolouch, it should all be "location-labels" everywhere

@dcalvin In that way, I'm afraid we need to update the PD section in the configuration file used by Ansible deployment. Because in that file, the configuration item is `location_labels`.

Reply:
Yes, in ansible is `location_labels`  but in pd api, it be `location-labels`.   :(

Reply:
@dcalvin Then, please update. Thanks!